pkg/cmd/roachtest/tests/rust_postgres.go

```diff
@@ -0,0 +1,168 @@
+// Copyright 2022 The Cockroach Authors.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package tests
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster"
+	"github.com/cockroachdb/cockroach/pkg/cmd/roachtest/option"
+	"github.com/cockroachdb/cockroach/pkg/cmd/roachtest/registry"
+	"github.com/cockroachdb/cockroach/pkg/cmd/roachtest/spec"
+	"github.com/cockroachdb/cockroach/pkg/cmd/roachtest/test"
+	"github.com/cockroachdb/cockroach/pkg/roachprod/install"
+)
+
+func registerRustPostgres(r registry.Registry) {
+	runRustPostgres := func(ctx context.Context, t test.Test, c cluster.Cluster) {
+		// Ask about this line
+		if c.IsLocal() {
+			t.Fatal("can not be run in local mode")
+		}
+		node := c.Node(1)
+		t.Status("setting up cockroach")
+		c.Put(ctx, t.Cockroach(), "./cockroach", c.All())
+		c.Start(ctx, t.L(), option.DefaultStartOpts(), install.MakeClusterSettings(), c.All())
+
+		version, err := fetchCockroachVersion(ctx, t.L(), c, node[0])
+		if err != nil {
+			t.Fatal(err)
+		}
+
+		if err := alterZoneConfigAndClusterSettings(ctx, t, version, c, node[0]); err != nil {
+			t.Fatal(err)
+		}
+
+		t.Status("cloning rust-postgres and installing prerequisites")
+		if err := repeatRunE(
+			ctx,
+			t,
+			c,
+			node,
+			"remove old rust-postgres",
+			`rm -rf /mnt/data1/rustpostgres`,
+		); err != nil {
+			t.Fatal(err)
+		}
+
+		if err := c.RunE(
+			ctx,
+			node,
+			"cd /mnt/data1 && git clone https://github.com/sfackler/rust-postgres.git",
+		); err != nil {
+			t.Fatal(err)
+		}
+		// TODO: Check if one is actually the option we want, may be 2
```

Comment:
what may be 2?

Reply:
Oop, this is another leftover comment from testing. Removing